build.gradle

```diff
@@ -298,7 +299,7 @@ dependencies {
     implementation 'com.google.guava:guava:30.0-jre'
     implementation 'org.greenrobot:eventbus:3.2.0'
     implementation 'commons-cli:commons-cli:1.3.1'
-    implementation 'org.bouncycastle:bcprov-jdk15on:1.67'
+    implementation "org.bouncycastle:bcprov-jdk18on:${bouncycastle_version}"
```

Comment:
Should we be matched with core for the bouncy castle version? Core has `bouncycastle` listed in `buildSrc/version.properties`

You can refer to these properties like `${versions.bouncycastle}`

Reply:
I will try to use a common version. Initially, I thought that the library with version postfix `jdk18on` should be used with Java 11. But this probably is not the case...

Reply:
Done, common version of Bouncy Castle used